ui/pages/_app.page.tsx

```diff
@@ -50,6 +50,7 @@ export default function App() {
 
       if (shouldRun) {
         if (NETWORK === "BERKELEY" || NETWORK === "LOCAL") {
+          await zkappWorkerClient.loadSnarkyJS();
```

Comment:
I added this line outside of this block earlier so that we load snarky before the user takes any action.  But I noticed we sometimes get an error related to calling stuff before it's loaded, so this should prevent that.  It won't take double as long.  If we get here and it already has been loaded, then it's a no-op.